tools/pfe-tools/config.ts

```diff
@@ -46,6 +46,7 @@ const SITE_DEFAULTS: Required<SiteOptions> = {
 const DEFAULT_CONFIG: PfeConfig = {
   demoURLPrefix: 'https://patternflyelements.org/',
   sourceControlURLPrefix: 'https://github.com/patternfly/patternfly-elements/tree/main/',
+  elementsDir: 'elements',
```

Comment:
I could see us needing this to be an array in the future.  I.e.  ["elements", "patterns", "apps"].  I think we could add this as a non-breaking feature in another release though.

Reply:
yeah probably. That would likely use some different kind of manifest though, a "pattern manifest", if you will.